packages/sdk/src/conversation/interface.ts

```diff
@@ -115,6 +115,26 @@ export interface NotificationOptions {
   storage?: NotificationTargetStorage;
 }
 
+/**
+ * The trigger pattern used to trigger a {@link TeamsFxBotCommandHandler} instance.
+ */
+export type TriggerPatterns = string | RegExp | (string | RegExp)[];
```

Comment:
Would it be simpler if just use `(string | RegExp)[]`? Support single element array as well.

Reply:
From developers' perspective, I think this would be more flexible, clear and straightforward if we also support `string` and `RegExp` in the surface, like the [hears() API in BotKit](https://github.com/howdyai/botkit/blob/main/packages/docs/reference/core.md#hears), although single element in array can also cover the case. 

Since this would not bring too much effort in our SDK development, why not just support it  :)